Automattic-Tracks-iOS/Model/Core Data/TracksContextManager.m

```diff
@@ -59,6 +74,51 @@ - (NSPersistentStoreCoordinator *)persistentStoreCoordinator {
     return _persistentStoreCoordinator;
 }
 
+- (NSURL *)storeURL {
+    return [[self storeContainerDirectoryURL] URLByAppendingPathComponent:@"Tracks.sqlite"];
+}
+
+- (NSURL *)storeContainerDirectoryURL {
+    if (self.sandboxed == YES) {
+        return [self applicationSupportURLForContainerApp];
+    } else {
+        return [self applicationDocumentsDirectory];
+    }
```

Comment:
@mokagio Is it possible this logic is inverted? (sandboxed > Documents Directory?).

Otherwise non sandboxed apps would still trigger the Documents Folder alert perhaps?

Reply:
You are totally right!

I didn't notice because the tests were inverted, too 🤦‍♂️